src/protocol/aggregation/mod.rs

```diff
@@ -21,19 +24,23 @@ use crate::{
         },
         BitDecomposed, Linear as LinearSecretSharing,
     },
+    seq_join::seq_join,
 };
 
 #[step]
 pub(crate) enum Step {
     Validator,
     ConvertValueBits,
+    ConvertBreakdownKeyBits,
+    ComputeEqualityChecks,
+    CheckTimesValue,
 }
 
-/// Binary-share aggregation protocol.
+/// Binary-share aggregation protocol for a sparse breakdown key vector input.
```

Comment:
Since this is the interface of this protocol, Can you also add a comment about the input and output format i.e. an array of [conversion value, breakdown key] and also output